core/vm/runtime/runtime.go

```diff
@@ -35,16 +35,21 @@ func (ruleSet) IsHomestead(*big.Int) bool { return true }
 func (ruleSet) IsAtlantis(*big.Int) bool  { return true }
 func (ruleSet) IsAgharta(*big.Int) bool   { return true }
 func (ruleSet) GasTable(*big.Int) *vm.GasTable {
+
+	//IsAgharta will always return true
+	//just have gastable default to returning the Agharta GasTable
 	return &vm.GasTable{
 		ExtcodeSize:     big.NewInt(700),
 		ExtcodeCopy:     big.NewInt(700),
+		ExtcodeHash:     big.NewInt(400),
 		Balance:         big.NewInt(400),
 		SLoad:           big.NewInt(200),
 		Calls:           big.NewInt(700),
 		Suicide:         big.NewInt(5000),
 		ExpByte:         big.NewInt(10),
```

Comment:
In the GasTable main.go, ExpByte is 50. Is it intentional that these are different?

Reply:
It's just what the gas cost was pre-byzantium. This gas table doesn't get used actually